regen-network/regen-web
# fix: APP-277 entering latitude, longitude in the location field

## Description

https://regennetwork.atlassian.net/browse/APP-277



---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...


### How to test

1. Create or edit a project. Go to the location form.
2. Type some coordinates in the format: "latitude, longitude".
3. You should see a reverse geocoding search result (only the city/place name gets displayed) and select it.
This will save the exact coordinates you entered but keep the found place name to display on the project page.
4. Create a post for this project and add a file with "no specific location (ie using the project location by default).
Check the map on the post page to confirm this is using the coordinates from step 2.

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...
